src/base/_base.scss

```diff
@@ -57,13 +57,13 @@ span.BtnSecondary {
   width: max-content;
   border-radius: 0;
   color: $color-black;
-  font-weight: 600;
+  font-weight: 400;
   box-sizing: border-box;
   line-height: 1;
   transition: all 400ms ease-in-out;
   font-family: $font-secondary;
   letter-spacing: 10%;
-  font-size: 1.25rem;
+  font-size: 1.25rem !important;
```

Comment:
Can you walk me through the reasoning for this important tag? Spoilers: I will probably always ask for this just because I hate important tags but for good reasons, I will accept them haha.

Reply:
Didn't mean to leave it there, stuck it in to see if it would change the font size on the shop by room button to help figure out (which it did not loll) what was causing the larger text size on it  and forgot to remove it!